src/main/kotlin/no/nav/syfo/dialogmote/tilgang/DialogmoteTilgangService.kt

```diff
@@ -23,6 +23,20 @@ class DialogmoteTilgangService(
         } else false
     }
 
+    suspend fun hasAccessToDialogmotePersonListWithObo(
+        personIdentNumberList: List<PersonIdentNumber>,
+        token: String,
+        callId: String,
+    ): List<PersonIdentNumber> {
+        return veilederTilgangskontrollClient.hasAccessToPersonListWithOBO(
+            personIdentNumberList = personIdentNumberList,
+            token = token,
+            callId = callId,
+        ).filter { personIdentNumber ->
+            !adressebeskyttelseClient.hasAdressebeskyttelse(personIdentNumber, token, callId)
```

Comment:
Må det gjøres noen tilpasninger i syfoperson også for at den skal virke med obo-token?

Reply:
Nå har jeg oppdatert til bruk av V2 av API'et til Syfoperson for adressebeskyttelse her.